Engine/Module/ModuleDebugDraw.cpp

```diff
@@ -393,6 +392,9 @@ bool ModuleDebugDraw::Init()
 
 void ModuleDebugDraw::Render()
 {
+#if GAME
```

Comment:
maybe instead of having this here, it is posible to just skip the instantiation of this module in Application class?

Reply:
Yeah, that can be done!